File: CB04.py
number_string = str(input())
number_a, number_b = map(int,number_string.split(" "))
print(number_a+number_b)
print(number_a-number_b)
print(number_a*number_b)
if number_b == 0:
  print("ERROR")
else:
  print(f"{round(number_a/number_b,2):.2f}")

File: CB04/CB04.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    string number_string;
    getline(cin, number_string);
    int number_a, number_b;
    sscanf(number_string.c_str(), "%d %d", &number_a, &number_b);
    cout << number_a + number_b << endl;
    cout << number_a - number_b << endl;
    cout << number_a * number_b << endl;
    if (number_b == 0) {
        cout << "ERROR" << endl;
    } else {
        cout << fixed << setprecision(2) << (double)number_a / number_b << endl;
    }
    return 0;
}
